qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -900,9 +900,9 @@ class _WebEnginePermissions(QObject):
     _options = {
         0: 'content.notifications',
         QWebEnginePage.Geolocation: 'content.geolocation',
-        QWebEnginePage.MediaAudioCapture: 'content.media_capture',
-        QWebEnginePage.MediaVideoCapture: 'content.media_capture',
-        QWebEnginePage.MediaAudioVideoCapture: 'content.media_capture',
+        QWebEnginePage.MediaAudioCapture: 'content.media.audio_capture',
+        QWebEnginePage.MediaVideoCapture: 'content.media.video_capture',
+        QWebEnginePage.MediaAudioVideoCapture: 'content.media.audio_video_capture',
```

Comment:
```suggestion
        QWebEnginePage.MediaAudioVideoCapture:
            'content.media.audio_video_capture',
```

to break the long line